__tests__/app.test.js

```diff
@@ -28,31 +28,32 @@ describe('app routes', () => {
       return client.end(done);
     });
 
-    test('returns animals', async() => {
+    test('returns todos', async() => {
 
       const expectation = [
         {
-          'id': 1,
-          'name': 'bessie',
-          'cool_factor': 3,
-          'owner_id': 1
+          id: 1,
+          todo: 'task1',
+          completed: false,
+          owner_id: 1
         },
         {
-          'id': 2,
-          'name': 'jumpy',
-          'cool_factor': 4,
-          'owner_id': 1
+          id: 2,
+          todo: 'task2',
+          completed: false,
+          owner_id: 1
         },
         {
-          'id': 3,
-          'name': 'spot',
-          'cool_factor': 10,
-          'owner_id': 1
+          id: 3,
+          todo: 'task3',
+          completed: false,
+          owner_id: 1
         }
       ];
+      
 
       const data = await fakeRequest(app)
-        .get('/animals')
+        .get('/todos')
```

Comment:
You'll need to add an authorization header to this route (and to the others when you get to them) - on the next line (between 56 and 57) you'll do `.set(Authorization: token)` to pass the token through.

Reply:
(this would be for the /api/todos route actually - which is the ultimate goal for what you'll be testing)